pkg/providers/vault/examples/nginx-pod-vault-inline-volume.yaml

```diff
@@ -18,7 +18,7 @@ spec:
         volumeAttributes:
           providerName: "vault"
           roleName: "example-role"
-          vaultAddress: "http://10.0.146.70:8200"
+          vaultAddress: "http://10.97.45.20:8200"
```

Comment:
Does this need to be updated?

Reply:
this is an accidental commit when i was testing, will revert